### DISCUSSION TITLE
Subplugin Template for tensor_decoder

### DISCUSSION THREAD

**Discussion by ayush6678** [nnstreamer/nnstreamer#4585] [Q&A] [answered]
Hi Maintainers, are there any subplugin template for tensor_decoder to extend support to new models like we have for tensor_filter ?

**Comment by myungjoo** [comment, marked as answer]
Interface: nnstreamer_plugin_api_decoder.h
There is no template, but you can use other subplugins as examples: /ext/nnstreamer/tensor_decoder
